homeassistant/components/fritzbox/light.py

```diff
@@ -118,7 +118,14 @@ def color_temp(self) -> int | None:
         return color.color_temperature_kelvin_to_mired(kelvin)
 
     @property
-    def supported_color_modes(self) -> set:
+    def color_mode(self) -> ColorMode:
+        """Return the color mode of the light."""
+        if self.device.color_mode == COLOR_MODE:
+            return ColorMode.HS
+        return ColorMode.COLOR_TEMP
```

Comment:
This looks straight forward, @mib1185 can you confirm it's correct though?

Reply:
Looks good.